### PR TITLE
feat: Myvotes placeholder

### LINKED ISSUE #208: Bug: My Votes Tab is Empty if you have never voted before
### Describe the bug

My Votes tab is empty if you have never voted before.

There could be a placeholder illustration there to say that this page is empty because you have not voted before or something

What do you think?

### Steps to reproduce

1. Go to [http://localhost:3000/](localhost)
2. Click on My Votes (If you have never voted before)
3. Page will be empty

### Screenshot
Screenshot 2022-07-15 235914

### Affected services

hot.opensauced.pizza

### Platforms

Desktop, Mobile

### Browsers

Chrome

### Environment

Development

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Deadreyo**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This puts a placeholder or an indication text in My Votes category when there are no voted repositories. This is instead of the empty list that was being displayed.

Other side fixes: Properly show skeletons for categories that are still loading (except Recent category)

## Related Tickets & Documents
closes #208 
## Mobile & Desktop Screenshots/Recordings
image

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: src/components/VotedRepoListWrap.tsx [hunk 1]
```diff
@@ -40,19 +40,36 @@ const VotedRepoListWrap = (): JSX.Element => {
...
+
+            <h1 className="text-2xl text-white font-semibold">
+              My Votes Repositories
+            </h1>
+          </div>
+
+          <div
+            className="p-4 border rounded-2xl bg-white w-full space-y-1 relative"
+          >
+            <h1 className="text-2xl text-center">You didn&apos;t vote on any Repositories. Your votes will appear here.</h1>
```
Comment (bdougie) [inline src/components/VotedRepoListWrap.tsx:58]: Not quite confident with this copy. My suggestion is a bit nicer for the viewers
```suggestion
            <h1 className="text-2xl text-center">Try voting on any repository. Your vote will appear here.</h1>
```

**Review by bdougie — CHANGES_REQUESTED** [review]
Just a copy fix to sound a bit nicer. Everything else looks good, except I am not sure why the HotRepositories were moved. I agree with the change, but it seemed unrelated.

File: src/components/VotedRepoListWrap.tsx [hunk 2]
```diff
@@ -40,19 +40,36 @@ const VotedRepoListWrap = (): JSX.Element => {
     setSearchParams({ limit: String(limit + 10) });
   };
 
+  if (!isLoading && !data.length) {
+    return (
+      <div className="mx-auto max-w-7xl px-4 pb-1 mt-10">
```
Comment (0-vortex) [inline src/components/VotedRepoListWrap.tsx:45]: the padding bottom added here looks great but it doesn't exist on the list repositories container; definitely no change when no voted repositories, but could use the padding on the list as well to make it look 💯

**Review by 0-vortex — CHANGES_REQUESTED** [review]
LGTM! 👍 

1 small change to make it perfect 🙏

**Review by bdougie — APPROVED** [review]
I am approving, but @0-vortex requested change is still pending.

**Comment by Deadreyo** [comment]
> LGTM! 👍
> 
> 1 small change to make it perfect 🙏

done

**Review by bdougie — APPROVED** [review]

**Pull request merged**